Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -15327,7 +15327,7 @@
         <ScoreMake value="100"/>
         <ScoreKill value="100"/>
         <ScoreResult value="BuildOrder"/>
-        <SubgroupPriority value="25"/>
+        <SubgroupPriority value="30"/>

```

Comment:
Havoc - spellcaster, maybe should be same as Arbiter as both of them rely on their passives